### PR TITLE
add Bulldozer eBee Backstory

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,6 +221,15 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
...
    </td>
   </tr>
+  <tr>
+   <td> <img src="images/bulldozer-bee.png" alt="Bulldozer">
+   </td>
+   <td>#26: Bulldozer eBee
+     <br />
+     <br />
+      Infrastructure is a constant work in progress. There is always something that needs to be built, replaced, upgraded, or renewed. Luckily, Bulldozer eBee in on the job. You'll often see him out in the field digging trenches for fiber or laying the groundwork for a new internet exchange point. Once he is finished, Electrician eBee is often right behind him to start up the real wiring.
```
Comment (qmonnet) [inline README.md:230]:
```suggestion
      Infrastructure is a constant work in progress. There is always something that needs to be built, replaced, upgraded, or renewed. Luckily, Bulldozer eBee in on the job. You'll often see him out in the field digging trenches for fiber or laying the groundwork for a new Internet exchange point. Once he is finished, Electrician eBee is often right behind him to start up the real wiring.
```

**Review by qmonnet — COMMENTED** [review]
Let's add a fun anecdote, too? One idea: “There's a persistant rumour circulating in the hive, stating that Bulldozer eBee would occasionally dig caches to store emergency honey supplies”.

File: README.md (hunk 1 above)
Comment (xmulligan) [inline README.md:230]:
```suggestion
      Infrastructure is a constant work in progress. There is always something that needs to be built, replaced, upgraded, or renewed. Luckily, Bulldozer eBee in on the job. You'll often see him out in the field digging trenches for fiber or laying the groundwork for a new Internet exchange point. Once he is finished, Electrician eBee is often right behind him to start up the real wiring. There is also a persistent rumour circulating in the hive that Bulldozer eBee occasionally digs caches to store emergency honey supplies.
```

**Comment by xmulligan** [comment]
Oh nice, I like that. Added

**Review by qmonnet — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -221,6 +221,15 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
...
    </td>
   </tr>
+  <tr>
+   <td> <img src="images/bulldozer-bee.png" alt="Bulldozer">
+   </td>
+   <td>#26: Bulldozer eBee
+     <br />
+     <br />
+      Infrastructure is a constant work in progress. There is always something that needs to be built, replaced, upgraded, or renewed. Luckily, Bulldozer eBee in on the job. You'll often see him out in the field digging trenches for fiber or laying the groundwork for a new Internet exchange point. Once he is finished, Electrician eBee is often right behind him to start up the real wiring. There is also a persistent rumour circulating in the hive that Bulldozer eBee occasionally digs caches to store emergency honey supplies.
```
Comment (kaworu) [inline README.md:230]: typo?
```suggestion
      Infrastructure is a constant work in progress. There is always something that needs to be built, replaced, upgraded, or renewed. Luckily, Bulldozer eBee is on the job. You'll often see him out in the field digging trenches for fiber or laying the groundwork for a new Internet exchange point. Once he is finished, Electrician eBee is often right behind him to start up the real wiring. There is also a persistent rumour circulating in the hive that Bulldozer eBee occasionally digs caches to store emergency honey supplies.
```

**Pull request merged**